js/description-photo.js

```diff
@@ -27,7 +27,7 @@ const createDescriptionPhoto = (index) => {
 
   return {
     id: descriptionId,
-    url: 'photos/' + descriptionId,
+    url: 'photos/' + descriptionId + '.jpg',
```

Comment:
красивее было бы template string
```suggestion
    url: `photos/${descriptionId}.jpg`,
```